test/unit/toggle_complete.ts

```diff
@@ -0,0 +1,142 @@
+import { expect } from "chai";
+import * as todoer from "../../src/core/todoer";
+
+describe("Completing a task", () => {
+  describe("Should toggle an incomplete task to complete", () => {
+    // prettier-ignore
```

Comment:
I was getting pretty frustrated with prettier.   Love hate relationship I guess.

I like it for things like automatically putting (or removing) semicolons, chaning double quotes to single quotes (or vice versa), automatically indending my lines and code blocks.  But I don't like that it reformats my strings , or arrays, etc.